src/new-components/stack/stack.module.css

```diff
@@ -3,42 +3,42 @@
     margin-top: calc(var(--reactist-spacing-xsmall) * -1);
```

Comment:
Just a question - why do we use a negative margin on the parent and then top margins/paddings in the children? Wouldn't it be easier to use bottom margins on all but the last child?

Reply:
We probably can. Also, it can still be top margin on all but the first child.

I went this way because on my early prototyping I did not manage something along those lines at first, and this worked. But it was probably a mishap of me not trying things in the right way, and since this worked first, I left it as is with the thought that this is internal implementation and that we could always revisit and improve it as long as we keep the behaviour.